HIP/hip-1028.md

```diff
@@ -0,0 +1,157 @@
+| --- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
+|0fcaca1f| `updateNFTsMetadata(address token, int64[] memory serialNumbers, bytes memory metadata)`                                                                                      | `int responsecode`                                                         |
+|ad7f8f0b| `createNonFungibleToken(HederaTokenV2 memory token)`                                                                                                                          | `(int responseCode, addess tokenAddress)`                                  |
+|c5bc16bc| `createNonFungibleTokenWithCustomFees(HederaTokenV2 memory token, FixedFee[] memory fixedFees,RoyaltyFee[] memory royaltyFees)`                                               | `(int responseCode, addess tokenAddress)`                                  |
+|7cb8323a| `createFungibleToken(HederaTokenV2 memory token, int64 initialTotalSupply, int32 decimals)`                                                                                   | `(int responseCode, addess tokenAddress)`                                  |
+|5ac3e67a| `createFungibleTokenWithCustomFees(HederaTokenV2 memory token, int64 initialTotalSupply, int32 decimals, FixedFee[] memory fixedFees, FractionalFee[] memory fractionalFees)` | `(int responseCode, addess tokenAddress)`                                  |
+|54c832a5| `updateTokenInfo(address token, HederaTokenV2 memory tokenInfo)`                                                                                                              | `int responseCode`                                                         |
+|bc03816f| `getTokenInfoV2(address token)`                                                                                                                                               | `(int64 responseCode, TokenInfoV2 memory tokenInfo)`                       |   
+|fb29ac6e| `getNonFungibleTokenInfoV2(address token, int64 serialNumber)`                                                                                                                | `(int64 responseCode, NonFungibleTokenInfoV2 memory nonFungibleTokenInfo)` |
+|3f9dc353| `getFungibleTokenInfoV2(address token)`                                                                                                                                       | `(int64 responseCode, FungibleTokenInfoV2 memory fungibleTokenInfo)`       |
+
+## Backwards Compatibility
+Backward compatibility is ensured by adding new functions where needed.
```

Comment:
Backwards compatibilty is by the consensus node supporting both the existing and the new methods (they've got different selectors).